INSTALL

```diff
@@ -86,9 +96,11 @@ line, which is needed. In order to make Qt available you need to
 add a symbol link (do something like
 ln -s <Qt location>/<version>/<type>/bin/qmake /usr/local/bin/qmake
 
-QtWebKit is needed, but no longer part of Qt5, so you need to download it and compile.
-We want to replace QtWebKit, but at the moment the print process depends on functions
-only available in QtWebKit
+QtWebKit is needed, if you want to print, but no longer part of Qt5,
+so you need to download it and compile. We want to replace QtWebKit,
+but at the moment the print process depends on functions only available
+in QtWebKit. In case you just want to test without print possibility
+omit this step.
```

Comment:
What would make sense is to add a few lines about how to install QtWebKit from source. Possibly in its own commit.

Reply:
Yes I planned on doing that in a separate commit.